cmd/notification.go

```diff
@@ -1054,6 +1054,32 @@ func (sys *NotificationSys) CPULoadInfo() []ServerCPULoadInfo {
 	return reply
 }
 
+// HardwareCPUInfo - CPU Hardware info
+func (sys *NotificationSys) HardwareCPUInfo() []ServerCPUHardwareInfo {
+	reply := make([]ServerCPUHardwareInfo, len(sys.peerClients))
+	var wg sync.WaitGroup
+	for i, client := range sys.peerClients {
+		if client == nil {
+			continue
+		}
+		wg.Add(1)
+		go func(client *peerRESTClient, idx int) {
+			defer wg.Done()
+			cpui, err := client.HardwareCPUInfo()
+			if err != nil {
+				reqInfo := (&logger.ReqInfo{}).AppendTags("remotePeer", client.host.String())
+				ctx := logger.SetReqInfo(context.Background(), reqInfo)
+				logger.LogIf(ctx, err)
+				cpui.Addr = client.host.String()
```

Comment:
We only set this in case of error, As far as the `Addr` is concerned we set the value of  `ServerCPUHardwareInfo` in `admin-server-info.go` with the value of `Addr`.  This sort of implementation is done in `MemUsageInfo`  and `CPULoadInfo`.